operator-rs/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "operator-rs"
```

Comment:
why we call everything -rs?

Reply:
we already have the go versions, but the packages themselves can be called the same thing u right